### PR TITLE
chore: cert update from island.is

### PR DESCRIPTION
**Pull request by OArnarsson**
See email from island.is below

> Ágæti viðtakandi,
> 
>  
> 
> Þessi póstur er sendur á alla skráða tengiliði notenda innskráningarþjónustu Stafræns Íslands.
> 
> Skilríki innskráningarþjónustunnar verða endurnýjuð að morgni 14. júní nk.
> 
>  
> 
> Eldra skilríki:
> 
> CN = Innskraning Island.is
> 
> SERIALNUMBER = 6503760649
> 
> OU = Auðkenning og undirritun
> 
> OU = Bunadarskilriki
> 
> C = IS
> 
> O = Þjóðskrá Íslands
> 
> OU = 20210702130510
> 
>  
> 
> Nýtt skilríki:
> 
> CN = Innskraning Island.is
> 
> SERIALNUMBER = 6503760649
> 
> OU = Auðkenning og undirritun
> 
> OU = Bunadarskilriki
> 
> C = IS
> 
> O = Þjóðskrá Íslands
> 
> OU = 20220516123320
> 
>  
> 
> Vinsamlegast gerið viðeigandi ráðstafanir vegna þessa ef þurfa þykir þannig að tryggt sé að þjónustustig skerðist ekki.
> 
> Nýju skilríkin má finna í viðhengi og á PEM formi hér að neðan:
> 
>  
> 
> -----BEGIN CERTIFICATE-----
> ...
> -----END CERTIFICATE-----
> 
> Tæknilegar upplýsingar um innskráningarþjónustu má finna hér. Ef þið þarfnist frekari upplýsinga, vinsamlegast hafið samband við island@island.is 
> 
> 
> 
> Bestu kveðjur,
> 
> Kristján Friðriksson

### CODE REVIEW

File: certificates/Milliskilriki.cer [hunk 1]
```diff
@@ -1,32 +1,30 @@
...
+BgEFBQcwAoY2aHR0cDovL2NkcC5pc2xhbmRzcm90LmlzL3NraWxyaWtpL2Z1bGxnaWx0YXVka2Vu
+bmkucDdiMAsGA1UdDwQEAwIF4DAfBgNVHSMEGDAWgBTCKT6G/4bE2jUfaaak/wGDPEozqTBDBgNV
+HR8EPDA6MDigNqA0hjJodHRwOi8vY3JsLmF1ZGtlbm5pLmlzL2Z1bGxnaWx0YXVka2VubmkvbGF0
+ZXN0LmNybDAdBgNVHQ4EFgQU0SOqN/DGELKdA++dg68quf14bNgwDQYJKoZIhvcNAQELBQADggEB
+AMiangTaGuXvtF/S5RYjEBLIqdknzRjwDA485w4lSisjKAGBN/6omUyj5h0Y6fCSDXyC0EeBdzVt
+CmdZl0DeKMpBiEP0SoL2t7xclrNZ+KBTqSZiD0Y7xBLVr0haKFi+ruJqRCHk8BNpV59M1XD121Fv
+KedLDLDu/Q4yq1GWo3ubEX2ezOoAgGYAfaarSYaXp4W/d2jJ3bMVsPCyYkj9JGZSNzHg68WhL7dU
+1RbQKSotibHUGVWNcwDnvcOovWAe4k5lWm/QtX+TwwWOtFaKmk7pok5jeIiiWIU+YSp8skkUCLwD
+bfDWYPFtBWkN4hQYQkg52Jyu6g90DAn19zgl/tM=
+
```
Comment (sunkan) [inline certificates/Milliskilriki.cer]: Can you remove the empty line since openssl can't parse it when it's there

**Review by sunkan — CHANGES_REQUESTED** [review]

File: certificates/Milliskilriki.cer (hunk 1 above)
Comment (OArnarsson) [reply certificates/Milliskilriki.cer]: Sure thing, thank you @sunkan

**Comment by sunkan** [comment]
@OArnarsson we are getting `Signature invalid!` when testing this.

Does this work for you?

**Comment by OArnarsson** [comment]
> @OArnarsson we are getting `Signature invalid!` when testing this.
> 
> Does this work for you?

Currently no. I think the update has not been done on their end.
All I know is that the email contained this cert and said they will update their systems this "morning" whatever that means 🤷 

> Skilríki innskráningarþjónustunnar verða endurnýjuð að morgni 14. júní nk.
**Rough translation:** The certs will be updated on the morning of June 14th.

I guess we'll start seeing the errors pile up once the update has taken place and then we'll know 👍

**Comment by sunkan** [comment]
Ok. It's a bit annoying that since we can't break all the logins by pushing this until they go live

**Comment by OArnarsson** [comment]
@sunkan Oh I know, this was only in preparation for the update. Sorry for the confusion.

**Pull request closed without merging**